mobile/views/Search/PeopleSearchView.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import * as React from 'react';
+import { Text } from 'react-native';
+import { withNavigation } from 'react-navigation';
+import compose from 'recompose/compose';
+import searchUsersQuery, {
+  type SearchUsersType,
+} from '../../../shared/graphql/queries/search/searchUsers';
+import viewNetworkHandler, {
+  type ViewNetworkHandlerProps,
+} from '../../components/ViewNetworkHandler';
+import Loading from '../../components/Loading';
+import { SearchView } from './style';
+import { UserListItem } from '../../components/Lists';
+import type { Navigation } from '../../utils/types';
+import InfiniteList from '../../components/InfiniteList';
+
+type Props = {
+  data: {
+    search: SearchUsersType,
+  },
+  ...$Exact<ViewNetworkHandlerProps>,
+  navigation: Navigation,
+};
+
+class UsersSearchView extends React.Component<Props> {
+  render() {
+    const { isLoading, data, navigation } = this.props;
+
+    if (data.search) {
+      const { search: { searchResultsConnection } } = data;
+      const hasResults =
+        searchResultsConnection && searchResultsConnection.edges.length > 0;
+      const results = hasResults
+        ? searchResultsConnection.edges.map(e => e && e.node)
+        : [];
+
+      return (
+        <SearchView>
+          {isLoading && <Loading />}
+          {!hasResults && <Text>No results</Text>}
+          {hasResults && (
+            <InfiniteList
+              data={results}
```

Comment:
Pagination?

Reply:
I haven't built out search pagination yet unfortunately, requires a bit more wiring up with Algolia - will handle this in the future :)